src/apps/companies/apps/referrals/send-referral/client/SendReferralForm.jsx

```diff
@@ -24,111 +22,76 @@ const SendReferralForm = ({
   companyContacts,
   companyName,
   companyId,
-  subject,
-  notes,
-  contact,
-  adviser,
   sendingAdviserTeamName,
   flashMessages,
-  progress = false,
-  formSubmitted = false,
-}) => {
-  useEffect(() => {
-    if (formSubmitted) {
-      addMessageWithBody(
-        'success',
-        'Referral sent.',
-        `You can <a href=${companies.referrals.list()}>see all of your referrals on your Homepage</a>.`
-      )
-      window.location.href = companies.detail(companyId)
-    }
-  }, [formSubmitted])
-  return (
-    <>
-      <LocalHeader
-        heading={'Send a referral'}
-        breadcrumbs={[
-          { link: dashboard(), text: 'Home' },
-          {
-            link: companies.index(),
-            text: 'Companies',
-          },
-          { link: companies.detail(companyId), text: companyName },
-          { text: 'Send a referral' },
-        ]}
-        flashMessages={flashMessages}
-      />
+}) => (
+  <>
+    <LocalHeader
+      heading={'Send a referral'}
+      breadcrumbs={[
+        { link: dashboard(), text: 'Home' },
+        {
+          link: companies.index(),
+          text: 'Companies',
+        },
+        { link: companies.detail(companyId), text: companyName },
+        { text: 'Send a referral' },
+      ]}
+      flashMessages={flashMessages}
+    />
 
-      <Main>
-        <Task>
-          {(getTask) => {
-            const saveTask = getTask(TASK_SAVE_REFERRAL, STATE_ID)
-            const openContactFormTask = getTask(
-              TASK_OPEN_REFERRALS_CONTACT_FORM,
-              STATE_ID
-            )
+    <Main>
+      <TaskForm
+        id={STATE_ID}
+        transformPayload={(values) => ({
+          values,
+          companyId,
+        })}
+        submissionTaskName={TASK_SAVE_REFERRAL}
+        analyticsFormName="send-referral-form"
```

Comment:
```suggestion
        analyticsFormName="sendReferralForm"
```

The performance analysts have asked for this prop to be in snake case in order to match their conventions.